python/ipywidgets/ipywidgets/widgets/tests/utils.py

```diff
@@ -1,11 +1,14 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-from ipykernel.comm import Comm
 from ipywidgets import Widget
 import ipywidgets.widgets.widget
 
-class DummyComm(Comm):
+import comm
+from ipykernel.comm import Comm
```

Comment:
```suggestion
```
import is not used right?

Reply:
It is used later in that file, in the teardown function for restoring the original Comm class